packages/hydrogen/src/createHydrogenContext.example.jsx

```diff
@@ -0,0 +1,91 @@
+import {createHydrogenContext} from '@shopify/hydrogen';
+import * as remixBuild from '@remix-run/dev/server-build';
+import {
+  createRequestHandler,
+  createCookieSessionStorage,
+} from '@shopify/remix-oxygen';
+
+export default {
+  async fetch(request, env, executionContext) {
+    const waitUntil = executionContext.waitUntil.bind(executionContext);
+    const [cache, session] = await Promise.all([
+      caches.open('hydrogen'),
+      AppSession.init(request, [env.SESSION_SECRET]),
+    ]);
+
+    /* Create context objects required to use Hydrogen with your credentials and options */
+    const hydrogenContext = createHydrogenContext({
+      /* Environment variables from the fetch function */
+      env,
+      /* Request object from the fetch function */
+      request,
+      /* Cache API instance */
+      cache,
+      /* Runtime utility in serverless environments */
+      waitUntil,
+      session,
+    });
+
+    const handleRequest = createRequestHandler({
+      build: remixBuild,
+      mode: process.env.NODE_ENV,
+      /* Inject the customer account client in the Remix context */
+      getLoadContext: () => ({...hydrogenContext}),
+    });
+
+    const response = await handleRequest(request);
+
+    if (session.isPending) {
+      response.headers.set('Set-Cookie', await session.commit());
+    }
+
+    return response;
+  },
+};
+
+class AppSession {
+  isPending = false;
```

Comment:
We actually do this in a few other places too. 

Without the local implementation of AppSession, the type wont pass.
But it was noted to be a smelly since it does indicate a non-complete example.

I can use @ts-ignore and just leave the example in-complete. Let me know how this feel, if folks are ok with it I will make this change in all the examples.
```
//@ts-ignore See example implementation in https://github.com/Shopify/hydrogen/blob/main/templates/skeleton/app/lib/session.ts
import {AppSession} from '~/lib/session';
```